diff --git a/node_modules/@react-navigation/bottom-tabs/lib/module/utils/useBottomTabBarHeight.js b/node_modules/@react-navigation/bottom-tabs/lib/module/utils/useBottomTabBarHeight.js
index 2d3eea8..e846db1 100644
--- a/node_modules/@react-navigation/bottom-tabs/lib/module/utils/useBottomTabBarHeight.js
+++ b/node_modules/@react-navigation/bottom-tabs/lib/module/utils/useBottomTabBarHeight.js
@@ -4,9 +4,9 @@ import * as React from 'react';
 import { BottomTabBarHeightContext } from "./BottomTabBarHeightContext.js";
 export function useBottomTabBarHeight() {
   const height = React.useContext(BottomTabBarHeightContext);
-  if (height === undefined) {
-    throw new Error("Couldn't find the bottom tab bar height. Are you inside a screen in Bottom Tab Navigator?");
-  }
+  // if (height === undefined) {
+  //   throw new Error("Couldn't find the bottom tab bar height. Are you inside a screen in Bottom Tab Navigator?");
+  // }
   return height;
 }
 //# sourceMappingURL=useBottomTabBarHeight.js.map
\ No newline at end of file
diff --git a/node_modules/@react-navigation/bottom-tabs/src/utils/useBottomTabBarHeight.tsx b/node_modules/@react-navigation/bottom-tabs/src/utils/useBottomTabBarHeight.tsx
index 3d917b2..dc82931 100644
--- a/node_modules/@react-navigation/bottom-tabs/src/utils/useBottomTabBarHeight.tsx
+++ b/node_modules/@react-navigation/bottom-tabs/src/utils/useBottomTabBarHeight.tsx
@@ -5,11 +5,11 @@ import { BottomTabBarHeightContext } from './BottomTabBarHeightContext';
 export function useBottomTabBarHeight() {
   const height = React.useContext(BottomTabBarHeightContext);
 
-  if (height === undefined) {
-    throw new Error(
-      "Couldn't find the bottom tab bar height. Are you inside a screen in Bottom Tab Navigator?"
-    );
-  }
+  // if (height === undefined) {
+  //   throw new Error(
+  //     "Couldn't find the bottom tab bar height. Are you inside a screen in Bottom Tab Navigator?"
+  //   );
+  // }
 
   return height;
 }
