diff --git a/node_modules/@sentry/react-native/RNSentry.podspec b/node_modules/@sentry/react-native/RNSentry.podspec
index e752ab8..d5f7558 100644
--- a/node_modules/@sentry/react-native/RNSentry.podspec
+++ b/node_modules/@sentry/react-native/RNSentry.podspec
@@ -28,7 +28,7 @@ Pod::Spec.new do |s|
   s.ios.deployment_target = "11.0"
   s.osx.deployment_target = "10.13"
   s.tvos.deployment_target = "11.0"
-  s.visionos.deployment_target = "1.0"
+  s.visionos.deployment_target = "1.0" if s.respond_to?(:visionos)
 
   s.preserve_paths = '*.js'
 
