diff --git a/node_modules/react-native-wagmi-charts/src/charts/line/Gradient.tsx b/node_modules/react-native-wagmi-charts/src/charts/line/Gradient.tsx
index 4c3d82f..904880b 100644
--- a/node_modules/react-native-wagmi-charts/src/charts/line/Gradient.tsx
+++ b/node_modules/react-native-wagmi-charts/src/charts/line/Gradient.tsx
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import Animated from 'react-native-reanimated';
+import Animated,{AnimatedProps} from 'react-native-reanimated';
 import { Defs, LinearGradient, Stop, Path, PathProps } from 'react-native-svg';
 
 import { LineChartDimensionsContext } from './Chart';
@@ -8,9 +8,10 @@ import useAnimatedPath from './useAnimatedPath';
 
 const AnimatedPath = Animated.createAnimatedComponent(Path);
 
-export type LineChartGradientProps = Animated.AnimateProps<PathProps> & {
+export type LineChartGradientProps = AnimatedProps<PathProps> & {
   color?: string;
   children?: React.ReactNode;
+  lastGradientValue?: number
 };
 
 let id = 0;
@@ -20,6 +21,7 @@ LineChartGradient.displayName = 'LineChartGradient';
 export function LineChartGradient({
   color: overrideColor = undefined,
   children,
+  lastGradientValue = 0.1,
   ...props
 }: LineChartGradientProps) {
   const { area } = React.useContext(LineChartDimensionsContext);
@@ -41,6 +43,8 @@ export function LineChartGradient({
 
   ////////////////////////////////////////////////
 
+
+
   return (
     <>
       {children ? (
@@ -65,9 +69,11 @@ export function LineChartGradient({
             y1="0"
             y2="100%"
           >
-            <Stop offset="20%" stopColor={color} stopOpacity={0.15} />
-            <Stop offset="40%" stopColor={color} stopOpacity={0.05} />
-            <Stop offset="100%" stopColor={color} stopOpacity={0} />
+              <Stop offset="10%" stopColor={color} stopOpacity={0.8} />
+              <Stop offset="20%" stopColor={color} stopOpacity={0.7} />
+              <Stop offset="40%" stopColor={color} stopOpacity={0.5} />
+              <Stop offset="70%" stopColor={color} stopOpacity={0.3} />
+              <Stop offset="98%" stopColor={color} stopOpacity={lastGradientValue} />
           </LinearGradient>
         </Defs>
       )}
