diff --git a/node_modules/react-native-safe-area-context/src/SafeAreaContext.tsx b/node_modules/react-native-safe-area-context/src/SafeAreaContext.tsx
index 87ad055..9e25a7e 100644
--- a/node_modules/react-native-safe-area-context/src/SafeAreaContext.tsx
+++ b/node_modules/react-native-safe-area-context/src/SafeAreaContext.tsx
@@ -150,7 +150,7 @@ export function withSafeAreaInsets<T>(
   return React.forwardRef<unknown, T>((props, ref) => {
     const insets = useSafeAreaInsets();
     return <WrappedComponent {...props} insets={insets} ref={ref} />;
-  });
+  }) as any;
 }
 
 /**
