diff --git a/node_modules/react-use-websocket/src/lib/util.ts b/node_modules/react-use-websocket/src/lib/util.ts
index 4113e7d..5b09a6a 100644
--- a/node_modules/react-use-websocket/src/lib/util.ts
+++ b/node_modules/react-use-websocket/src/lib/util.ts
@@ -5,7 +5,7 @@ import { resetSubscribers } from './manage-subscribers';
 export function assertIsWebSocket (
     webSocketInstance: WebSocketLike,
 ): asserts webSocketInstance is WebSocket {
-    if (webSocketInstance instanceof WebSocket === false) throw new Error('');
+    if (webSocketInstance instanceof WebSocket === false) throw new Error('AssertIsWebSocket - Safe to ignore');
 };
 
 
