diff --git a/node_modules/react-native-svg/lib/typescript/elements/Path.d.ts b/node_modules/react-native-svg/lib/typescript/elements/Path.d.ts
index 03b93d9..bde3331 100644
--- a/node_modules/react-native-svg/lib/typescript/elements/Path.d.ts
+++ b/node_modules/react-native-svg/lib/typescript/elements/Path.d.ts
@@ -4,6 +4,7 @@ import type { CommonPathProps, NumberProp } from '../lib/extract/types';
 export interface PathProps extends CommonPathProps {
     d?: string;
     opacity?: NumberProp;
+    lastGradientValue?: number
 }
 export default class Path extends Shape<PathProps> {
     static displayName: string;
diff --git a/node_modules/react-native-svg/src/elements/Path.tsx b/node_modules/react-native-svg/src/elements/Path.tsx
index ff2ab70..03a32b5 100644
--- a/node_modules/react-native-svg/src/elements/Path.tsx
+++ b/node_modules/react-native-svg/src/elements/Path.tsx
@@ -8,6 +8,7 @@ import type { NativeMethods } from 'react-native';
 export interface PathProps extends CommonPathProps {
   d?: string;
   opacity?: NumberProp;
+  lastGradientValue?: number;
 }
 
 export default class Path extends Shape<PathProps> {
